src/main/java/hudson/maven/reporters/AbstractMavenJavadocArchiver.java

```diff
@@ -46,12 +46,6 @@
  */
 public abstract class AbstractMavenJavadocArchiver extends MavenReporter {
 
-
-    private boolean aggregated = false;
-
-    private FilePath target;
```

Comment:
Besides it being wrong to save a `FilePath` via XStream, this code was potentially very wrong from the start: a publisher should never mutate its own state in the course of a build! That could lead to one build contaminating results from another.

In this case, that would not be a problem in the usual case of `MavenReporterDescriptor.newAutoInstance`, but could be if the Javadoc archiver is explicitly configured in `MavenModuleSet.reporters`.